metaflow/metaflow_config.py

```diff
@@ -36,8 +36,50 @@ def init_config():
 METAFLOW_CONFIG = init_config()
 
 
-def from_conf(name, default=None):
-    return os.environ.get(name, METAFLOW_CONFIG.get(name, default))
+def from_conf(name, default=None, from_env_only=False, validate_fn=None):
```

Comment:
from_env_only does not seem to be used here (or I missed it), what is the use case?

Reply:
I'll remove this. It was used for a security token value  ("shared access signature") that should not live in the config JSON - but we decided not to support signature at all.